lte/protos/apn.proto

```diff
@@ -80,17 +82,15 @@ message APNConfiguration {
 // Network wide APN configuration
 message APNConfig {
   uint32 id = 1;
-  // Contains either the APN Name or wildcard "*"
-  string service_selection = 2;
-  QoSProfile qos_profile = 3;
-  AggregatedMaximumBitrate ambr = 4;
+  QoSProfile qos_profile = 2;
+  AggregatedMaximumBitrate ambr = 3;
   enum PDNType {
       IPV4 = 0;
       IPV6 = 1;
       IPV4V6 = 2;
       IPV4_OR_IPV6 = 3;
   }
-  PDNType pdn = 5;
+  PDNType pdn = 4;
```

Comment:
we do not renumber the legacy fields for backwards compatibility. use deprecated for fields that are no longer used.

Reply:
This is not yet being used. This is part of the new proto struct I'm adding as a part of the feature. Hence did the renumbering.